packages/flutter_tools/lib/src/dart/pub.dart

```diff
@@ -243,18 +243,6 @@ class _DefaultPub implements Pub {
         'The timestamp was: $originalPubspecYamlModificationTime\n'
         'The time now is: $now'
       );
-    } else {
```

Comment:
Update comment starting on 233. Also could you add an explanation in the PR description about why we don't actually rely on .packages having a more recent modification time.

Reply:
Done